src/components/Label/Label.jsx

```diff
@@ -3,48 +3,52 @@ import React, { type Node } from 'react';
 import cx from 'classnames';
 import { getDataAttrs } from '../../dataUtils';
 import type { Data } from '../../types';
-import style from './style.scss';
-
-const types = {
-  PRIMARY: 'primary',
-  SECONDARY: 'secondary',
-  DISABLED: 'disabled',
-};
+import styles from './style.scss';
 
 type Props = {|
-  text: string,
-  type: string,
```

Comment:
The `type` isn't mentioned in Plasma design documentation. However, removing the `type` prop is breaking changes.

Reply:
Yep. The plan was to fix all the breaking changes in SS when upgrading its Plasma dependency.